source/_components/ebusd.markdown

```diff
@@ -0,0 +1,119 @@
+---
+layout: page
+title: "ebusd"
+description: "The ebusd component allows the integration between eBUS heating system and Home Assistant."
+date: 2018-12-27 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ebusd.png
+ha_category: Hub
```

Comment:
We've had some discussion internally about the "Hub" category.
Conclusion: What "Hub" should be is when another Home Automation system/hub integrates with Home Assistant (e.g., Samsung SmartThings). While there are many things in Hub that should be moved out of there (which we started working on already), we should not add to it.

This component provides a Sensor, so that is what it should be categorized in.

```suggestion
ha_category: Sensor
```